src/components/Resource.js

```diff
@@ -128,6 +136,23 @@ export default class Resource extends React.Component {
     }
   }
 
+  isUpdatedResource(id) {
+    this.state.resources.forEach((i) => {
```

Comment:
can you change the name of the variable to `resource`? `i` is normally reserved for an int index